### ISSUE TITLE
Add support for a custom list of domains

### ISSUE THREAD

**Issue by NazarenoL** [barbushin/javascript-errors-notifier#90]
I would like to set this to trigger notification windows when using the production/canary versions of the site I work for, as well as my custom local urls.

Ideally, I could set a list of domains (_"www.site.com,test.site.local"_) to override the extension's detection of localhosts.

**Comment by andre-ellrich-adup** [comment]
yes, I need this as well. It would be great to have a list of regexes where the plugin would be active!
